accelerator/config.c

```diff
@@ -89,6 +89,9 @@ static status_t cli_core_set(char* conf_file, ta_config_t* const ta_conf, iota_c
     case CACHE:
       cache->cache_state = (toupper(value[0]) == 'T');
       break;
+    case PROXY_API:
+      ta_conf->proxy_passthrough = (toupper(value[0]) == 'T');
```

Comment:
The default value of `proxy_passthrough` is `true`.
And it will be set as `false` with CLI option `--proxy_api *` exclude `--proxy_api t*`?

Reply:
I will set the var into the opposite value